### ISSUE TITLE
Issues found testing with TalkBack screenreader

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/talk-android#222]
I did some quick testing with the [TalkBack screenreader](https://support.google.com/accessibility/android/answer/6283677) which is included in the Android accessibility settings and here’s some basic issues I found:

- [ ] In the conversation list, the 3-dot-menu button is "unlabeled button", should be "Menu for conversation with Ivan" or "Menu for group conversation Company call"
- [ ] Send button is "unlabeled button", should be "Send message"

What do you think @nextcloud/accessibility @henningoschwald? cc @mario

**Comment by mario** [comment]
Done. Let me know if you find something else.

**Comment by jancborchardt** [comment]
Thank you! Also do test it yourself if there's new additions or views. :)
